packages/story-editor/src/components/panels/design/preset/stories/confirmationDialog.js

```diff
@@ -24,7 +24,7 @@ import { boolean } from '@storybook/addon-knobs';
  * Internal dependencies
  */
 import ConfirmationDialog from '../confirmationDialog';
```

Comment:
Note: More refactoring can be done here for the style presets, this will be part of #9463